### PR TITLE
Move handling of VISION_POSITION_DELTA mavlink message up

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: AntennaTracker/make.inc [hunk 1]
```diff
@@ -33,3 +33,4 @@ LIBRARIES += AP_RSSI
 LIBRARIES += AP_RPM
 LIBRARIES += AP_Frsky_Telem
 LIBRARIES += AP_Beacon
+LIBRARIES += AP_VisualOdom
```
[the remark this answers is not in the document]
Comment (peterbarker) [reply AntennaTracker/make.inc]: Fixed.  I'd forgotten / wasn't aware of that file.  My guess is there's other common libraries that should be moved into there.

**Comment by peterbarker** [comment]
```
      4 LIBRARIES += AC_AttitudeControl
      4 LIBRARIES += AC_Avoidance
      4 LIBRARIES += AC_Fence
      4 LIBRARIES += AP_Arming
      4 LIBRARIES += AP_Buffer
      4 LIBRARIES += AP_Camera
      4 LIBRARIES += AP_Common
      4 LIBRARIES += AP_Proximity
      4 LIBRARIES += AP_RCMapper
      4 LIBRARIES += AP_RSSI
      5 LIBRARIES += AC_PID
      5 LIBRARIES += AP_AccelCal
      5 LIBRARIES += AP_AHRS
      5 LIBRARIES += AP_Baro
      5 LIBRARIES += AP_BattMonitor
      5 LIBRARIES += AP_Beacon
      5 LIBRARIES += AP_BoardConfig
      5 LIBRARIES += AP_Compass
      5 LIBRARIES += AP_Declination
      5 LIBRARIES += AP_Frsky_Telem
      5 LIBRARIES += AP_GPS
      5 LIBRARIES += AP_InertialSensor
      5 LIBRARIES += AP_Math
      5 LIBRARIES += AP_Mission
      5 LIBRARIES += AP_Mount
      5 LIBRARIES += AP_NavEKF2
      5 LIBRARIES += AP_NavEKF3
      5 LIBRARIES += AP_Notify
      5 LIBRARIES += AP_OpticalFlow
      5 LIBRARIES += AP_Param
      5 LIBRARIES += AP_Rally
      5 LIBRARIES += AP_RangeFinder
      5 LIBRARIES += AP_RPM
      5 LIBRARIES += AP_Scheduler
      5 LIBRARIES += AP_SerialManager
      5 LIBRARIES += AP_Terrain
      5 LIBRARIES += AP_Vehicle
      5 LIBRARIES += DataFlash
      5 LIBRARIES += Filter
      5 LIBRARIES += GCS_MAVLink
      5 LIBRARIES += RC_Channel
      5 LIBRARIES += StorageManager
```

(`pbarker@bluebottle:~/rc/ardupilot/Tools/autotest/param_metadata(visual-odo-up)$ cat ~/rc/ardupilot/*/make.inc | grep LIBRARIES | sort | uniq -c | sort`)

**Comment by peterbarker** [comment]
@OXINARF First commit removed.

**Comment by rmackay9** [comment]
I think peter addressed @OXINARF's issues so merging.  So to rush, i'm just quite keen on this and the EKF external-position-stuff going in because there is a line of people waiting for it.

**Pull request merged**